ui/StatusQ/src/StatusQ/Components/StatusMessageHeader.qml

```diff
@@ -61,82 +59,168 @@ Item {
                     root.clicked(this, mouse)
                 }
             }
-        }       
-        StatusBaseText {
-            id: messageOriginInfo
-            verticalAlignment: Text.AlignVCenter
-            visible: text
-            color: Theme.palette.baseColor1
-            font.pixelSize: Theme.asideTextFontSize
-            text: root.messageOriginInfo
         }
-        StatusContactVerificationIcons {
-            id: verificationIcons
-            visible: !root.amISender
-            isContact: root.isContact
-            trustIndicator: root.trustIndicator
+
+        Loader {
```

Comment:
Putting everything into Loader here feels too granular. Wouldn't it be better to simply load `StatusMessageHeader` lazily and asynchronously? I believe there wouldn't be much difference in terms of performance, would it?

Reply:
The loaders are added on the components that have `visible` property bindings and replaced the visible binding with Loader.active. Usually these bindings won't change in the lifetime of the message and I'm not sure it's worth creating the items just to hide them.